aea/cli/run.py

```diff
@@ -214,73 +240,34 @@ def _print_instantiated_components(aea: AEA) -> None:
             )
         ]
 
-    max_col_1_length = max(list(map(len, components)))
-    table_width = max_col_1_length + 6
-    row_separator = "=" * table_width
-    padding = " " * 2
-
-    def format_row(col_1: str) -> str:
-        """Format a row."""
-        return (
-            "|"
-            + padding
-            + col_1
-            + " " * (max_col_1_length - len(col_1))
-            + padding
-            + "|"
-        )
-
-    click.echo("Intantiated packages.")
-    click.echo(row_separator)
-    click.echo(format_row("ComponentId"))
-    click.echo(row_separator)
-    for component_id in components:
-        click.echo(format_row(component_id))
-    click.echo(row_separator)
-    click.echo()
-    click.echo()
+    click.echo("All instantiated components")
+    print_table([(c,) for c in components])
 
 
 def _print_all_available_packages(ctx: Context) -> None:
     """Print hashes for all available packages"""
     ipfs_hash = IPFSHashOnly()
-    max_col_1_length = 0
-    max_col_2_length = 48
-
-    rows = []
+    rows = [("Package", "IPFSHash")]
 
     for package_id, package_path in list_available_packages(ctx.cwd):
         package_hash = ipfs_hash.hash_directory(str(package_path), wrap=False)
         rows.append((str(package_id), package_hash))
-        max_col_1_length = max(max_col_1_length, len(str(package_id)))
-
-    table_width = max_col_2_length + max_col_1_length + 9
-    row_separator = "=" * table_width
-    padding = " " * 2
-
-    def format_row(col_1: str, col_2: str) -> str:
-        """Format a row."""
-        return (
-            "|"
-            + padding
-            + col_1
-            + " " * (max_col_1_length - len(col_1))
-            + "|"
-            + padding
-            + col_2
-            + " " * (max_col_2_length - len(col_2))
-            + padding
-            + "|"
-        )
 
-    click.echo("Available packages.")
-    click.echo(row_separator)
-    click.echo(format_row("Package", "IPFSHash"))
-    click.echo(row_separator)
-    for row in rows:
-        click.echo(format_row(*row))
-    click.echo(row_separator)
-    click.echo()
+    click.echo("All available packages.")
+    print_table(rows)
+
+
+def _print_addresses(aea: AEA) -> None:
+    """Print all the addresses used by agent."""
+
+    addresses = [
+        ("Name", "Address"),
+        *((k, v) for k, v in aea.context.addresses.items()),
+        ("decision_maker", aea.context.decision_maker_address),
+    ]
```

Comment:
@DavidMinarsch Are there any other addresses we should be printing?

Reply:
We don't need the decision_maker one. Otherwise this is good.